handcrafts.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Products Landing Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <div class = "menu">
+            <div class="logo">
+                <img src="image/product-landing-page-logo.png" width="300px" alt="logo">
+            </div>            
+            <div class="menu_items">
+                <a href="#">Features</a></li>
+                <a href="#">How It Works</a>
+                <a href="#">Pricing</a>
+            </div>
+            
+        </div>
+        <div class="title">
+            <h2>Handcrafted, home-made masterpieces</h2>
+            <br>
+            <input type="text" placeholder="Enter your email address">
+            <br>
+            <br>
+            <input type="submit" value ="GET STARTED">
```

Comment:
It is better to use `<section>` rather than `<div>` when starting a new independent section.